torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -3020,6 +3020,40 @@ RegisterOperators reg2({
           return 0;
         },
         aliasAnalysisFromSchema()),
+    Operator(
+        "aten::all(int[] self) -> bool",
```

Comment:
Could you add `float[]` as well ?

Reply:
Float has been added in the latest commit.